cli/command_policy_set_retention.go

```diff
@@ -40,11 +42,25 @@ func (c *policyRetentionFlags) setRetentionPolicyFromFlags(ctx context.Context,
 		{"number of latest backups to keep", &rp.KeepLatest, c.policySetKeepLatest},
 	}
 
-	for _, c := range cases {
+	for _, c := range intCases {
 		if err := applyOptionalInt(ctx, c.desc, c.max, c.flagValue, changeCount); err != nil {
 			return err
 		}
 	}
 
+	boolCases := []struct {
```

Comment:
the loop is not needed for now since it runs once, I would just call applyPolicyBoolPtr